app/src/main/java/com/nononsenseapps/feeder/model/opml/OPMLImporter.kt

```diff
@@ -25,12 +27,22 @@ open class OPMLImporter(override val di: DI) : OPMLParserHandler, DIAware {
 
     override suspend fun saveFeed(feed: Feed) {
         val existing = feedDao.loadFeedWithUrl(feed.url)
-
+        val existingNostrFeeds = feedDao.getAllFeeds().filter { it.url.isAdaptedUrlFromNostrUri() }
```

Comment:
Don't load all feeds here for every feed.

Reply:
Fixed.